docs/index.rst

```diff
@@ -22,24 +22,25 @@ Dive in!
    :maxdepth: 2
    :caption: Getting Started
 
-   Installation <installation>
+   Installation <getting_started/installation>
 
 .. add basic usage example here
 
 .. toctree::
-   :maxdepth: 2
+   :maxdepth: 1
    :caption: Ease of Use with a Real Robot
    
-   Simulation vs Real Robot <simvsreal>
-   Interface with a Real Robot <driverinterface>
-   Timestep Logic <timesteplogic>
+   Switching between the Real Robot and the Simulated Robot in this Simulation <simreal/simwithreal>
```

Comment:
From what I saw, the titles here are the same as the titles of the actual files.  In this case, you don't need to repeat it here but you can simply list the filename.

```suggestion
   simreal/simwithreal
```